client/src/components/layout/UpdateReviewForm.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react"
+import { Redirect } from "react-router-dom"
+import ErrorList from "./ErrorList.js"
+import translateServerErrors from "../../services/translateServerErrors.js"
+
+const updateReviewForm = (props) => {
+  const [updatedReview, setUpdatedReview] = useState({
+    body: "",
+    rating: ""
+  })
+
```

Comment:
this component will have a useEffect fetch request to retrieve the Review information, given that it is on its own page

Reply:
that existing review's info should be fetched from the reviewsRouter.get("/:id")